docs/blog/hello-world.md

```diff
@@ -0,0 +1,8 @@
+# Hello Microsoft MakeCode!
+
+### Posted on 3/14/2017
```

Comment:
I guess ``**Posted on ...**`` would make more sense from document structuring standpoint.

Also, March 14, 2017 or 2017-03-14 would be less ambiguous (for future dates).